tests/common/ixia/ixia_fixtures.py

```diff
@@ -0,0 +1,141 @@
+        Ixia PTF server (API server) IP
+    """
+    return testbed['ptf_ip']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_user(duthost):
+    """
+    Return the username of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) username.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['user']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_passwd(duthost):
+    """
+    Return the password of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) password.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['password']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_port(duthost):
+    """
+    Return REST port of the ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) REST port.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['rest_port']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_session_id(duthost):
+    """
+    Ixia PTF can spawn multiple session on the same REST port.
+    Optional for LINUX, required for windows return the session ID.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+
+    Returns:
+        Ixia PTF server (API server) session id.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['session_id']
+
+
+@pytest.fixture(scope = "module")
+def ixia_dev(duthost, fanouthosts):
+    """
+    Returns the Ixia chassis IP
+
+    Args:
+        duthost (pytest fixture): The duthost fixture. 
+        fanouthosts (pytest fixture): The fanouthosts fixture.
+
+    Returns:
+        Ixia Chassis IP.
+    """
+    result = dict()
+    ixia_dev_hostnames = fanouthosts.keys()
+    for hostname in ixia_dev_hostnames:
+        result[hostname] = duthost.host.options['inventory_manager'].get_host(hostname).get_vars()['ansible_host']
+    return result
+
+
+@pytest.fixture(scope = "function")
+def ixia_api_server_session(ixia_api_serv_ip,
+    ixia_api_serv_user, ixia_api_serv_passwd, ixia_api_serv_port,
+    ixia_api_serv_session_id) :
+    """
+    Ixia session manager fixture.
+
+    Args:
+        ixia_api_serv_ip (pytest fixture): ixia_api_serv_ip fixture
+        ixia_api_serv_user (pytest fixture): ixia_api_serv_user fixture.
+        ixia_api_serv_passwd (pytest fixture): ixia_api_serv_passwd fixture.
+        ixia_api_serv_port (pytest fixture): ixia_api_serv_port fixture.
+        ixia_api_serv_session_id (pytest fixture): ixia_api_serv_session_id 
+            fixture.
+  
+    Returns:
```

Comment:
Returns -> Return

Reply:
Google style docstring recommends “Returns” instead of “Return”(https://google.github.io/styleguide/pyguide.html). So, I kept it as it is.